knowledge_runtime/tests/test_config_resolver.py

```
@@ -0,0 +1,1104 @@
+                db=mock_db,
+                user_id=42,
+                name="test-retriever",
+                namespace="default",
+            )
+
+        assert isinstance(result, RuntimeRetrieverConfig)
+        assert result.name == "test-retriever"
+        assert result.namespace == "default"
+        assert result.storage_config["type"] == "qdrant"
+        assert result.storage_config["url"] == "http://localhost:6333"
+        assert result.storage_config["password"] == "decrypted_enc_password"
```

Ruff (0.15.11)

[error] Possible hardcoded password assigned to: "password"

(S105)